src/modules/Dropdown/Dropdown.js

```diff
@@ -848,12 +861,12 @@ export default class Dropdown extends Component {
   // Setters
   // ----------------------------------------
 
-  setValue = (value) => {
+  setValue = (value, resetQuery = true) => {
```

Comment:
I think that we should extract behaviour of `removeItemOnBackspace` to separate issue, I fully agree that we need to impliment it.